addons/default/visiosoft/profile-module/src/Seed/UsersFieldsSeeder.php

```diff
@@ -124,21 +124,31 @@ public function run(
                     "picker" => true,
                 ]
             ],
-            [
-                'name' => trans('visiosoft.module.profile::field.education.name'),
-                'slug' => 'education',
-                'type' => 'anomaly.field_type.text',
-            ],
-            [
-                'name' => trans('visiosoft.module.profile::field.state_of_education.name'),
-                'slug' => 'state_of_education',
-                'type' => 'anomaly.field_type.text',
-            ],
-            [
-                'name' => trans('visiosoft.module.profile::field.profession.name'),
-                'slug' => 'profession',
-                'type' => 'anomaly.field_type.text',
-            ],
+	        [
+		        'name' => trans('visiosoft.module.profile::field.education.name'),
+		        'slug' => 'education',
+		        'type' => 'anomaly.field_type.text',
+	        ],
+	        [
+		        'name' => trans('visiosoft.module.profile::field.state_of_education.name'),
+		        'slug' => 'state_of_education',
+		        'type' => 'anomaly.field_type.text',
+	        ],
+	        [
+		        'name' => trans('visiosoft.module.profile::field.profession.name'),
+		        'slug' => 'profession',
+		        'type' => 'anomaly.field_type.select',
+	        ],
+	        [
+		        'name' => trans('visiosoft.module.profile::field.education_part.name'),
+		        'slug' => 'education_part',
+		        'type' => 'anomaly.field_type.select',
+	        ],
+	        [
+		        'name' => trans('visiosoft.module.profile::field.education_part_option.name'),
+		        'slug' => 'education_part_option',
+		        'type' => 'anomaly.field_type.select',
+	        ],
```

Comment:
are we using this field "education_part_option"? or any of these fields in that matter? can you recheck the fields so we don't have unused fields